### PR TITLE
AR-893: add four areas screen template

### PR DESCRIPTION
**Pull request by sinejespersen**
[ git push --set-upstream origin feature/AR-893-add-four-area-screen-template
](https://jira.itkdev.dk/browse/AR-893)

image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ All notable changes to this project will be documented in this file.
 
 ## [1.2.0] - 2023-01-05
 
+- [#101](sdf)
```
Comment (sinejespersen) [inline CHANGELOG.md:9]:
```suggestion
- [#101](https://github.com/os2display/display-templates/pull/102)
```

File: src/screen-layouts/four-areas.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "title": "Seks områder",
```
Comment (sinejespersen) [inline src/screen-layouts/four-areas.json:2]:
```suggestion
  "title": "Fire områder",
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ All notable changes to this project will be documented in this file.
 
 ## [1.2.0] - 2023-01-05
 
+- [#101](https://github.com/os2display/display-templates/pull/102)
```
Comment (tuj) [inline CHANGELOG.md:9]: #102

**Review by tuj — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (sinejespersen) [reply CHANGELOG.md]: fixed

**Pull request merged**